src/data/pokeball.ts

```diff
@@ -57,6 +58,12 @@ export function getPokeballCatchMultiplier(type: PokeballType): number {
   case PokeballType.ULTRA_BALL:
     return 2;
   case PokeballType.ROGUE_BALL:
+    /* making rogue balls have a higher chance to catch pokemon if they have a tinted pokeball (i.e. at least one thing is new for them)
+       * you can also get the boosted type for a description (i.e. for modifier-type)
+       */
```

Comment:
```suggestion
    // Rogue balls have a higher chance to catch pokemon if they have a tinted pokeball icon (i.e. at least one thing is new for them).
    // You can also get the boosted type for a description (i.e. for modifier-type).
```
> You can also get the boosted type for a description (i.e. for modifier-type).

What exactly does this mean?

Reply:
The boosted type for the description in shops. Since normally it's just listed as "Catch rate: 3x", there needs to be a way to get the boosted value without a pokemon. The getBoostedDescription boolean does that, so if that's passed, then the it's for the shop description.